### PR TITLE
Remove service specific topic logic

### PR DESCRIPTION
**Pull request by dehort**
## Why?
- The service specific topics are no longer used.  Remove the old logic to simplify the code a bit.
- Rename config option from VALIDTOPICS to VALID_UPLOAD_TYPES (is this a good name?)
- Removed "hidden" mapping of "openshift" to "buckit" inside ingress service (added "openshift" to VALID_UPLOAD_TYPES)

## How?
Describe how the change is implemented. Any noteable new libaries, APIs, or features.

## Testing
Did you add any tests for the change?

## Anything Else?
Any other notes about the PR that would be useful for the reviewer. 

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,18 +26,11 @@ The Ingress workflow is as follows:
   cloud storage, and puts a message on the announcement topic while applying a header
   to identify the destination service.
 
-For the vast majority of topics, the above process is accurate. For payloads from both
+For the vast majority of upload types, the above process is accurate. For payloads from both
 **ansible tower** and the **insights cluster operator**, the flow is a bit different. The key difference here is that it relies on the UHC Auth Proxy for authentication.
 
 ![UML](https://www.plantuml.com/plantuml/png/TP5FSvim4CNl-HHRNzBEDEP0Jpt5mTF6qwRrXFYSMSC6D0Y9QbTntKzV2Mpe4FV4_dc_vV6uPK4dljLNxvGfj2y9Qf6EFoU9myEoKbBxlMToXJL2HfQ5RPDEeudC3KkfrJx9FjriusZty3rfaOLS63rdWK1bo2sxUFygFuPD-pwy92e-mY8RgaKeDuPLLGl3puuSYdMB3sSzDjYY2ffLNqHrjlunxLCkK5EOfdaiudwrtS1N53hWSTBvkWYhtNq6AoyrR9tzVOpYs94HLQ0eQo0dzweAcZXbAaVCqUHGHMZNQOlbMp6BTLZHdRDD_udvqCCmY6IUdfeBSDhlUrCj_h46xhGj6ZWVcO17qbEEOq23gTxV_TFJDX-4puzZX6DKNwmxe2jXZqmbM0Daoiug8pDs33U6Duzg9Xbp6g-BFG-11-lpyoF3wHHgXyVuZ3WBLa43Uryq9F-dvwcJAQ4Dgp2CPzx-XM_uqk3fp8pkhJpOL_hNoBLrrMQTd38FbQDVdbWswsjG1cn7Xclr8XUStWOpljL_0G00)
 
-### Kafka Topics (Legacy)
```
Comment (gmcculloug) [inline README.md:34]: Along with this you also need to remove the reference to "legacy" in the `Content Type` section:
> In the case of the legacy operation, the service name tells ingress what topic to use:
>
>application/vnd.redhat.advisor.example+tgz => platform.upload.advisor

File: deploy/clowdapp.yaml [hunk 2]
```diff
@@ -137,8 +137,8 @@ objects:
 parameters:
 - name: INGRESS_STAGEBUCKET
   value: insights-upload-perma
-- name: INGRESS_VALIDTOPICS
-  value: advisor,compliance,hccm,qpc,rhv,tower,leapp-reporting,buckit,xavier,mkt,playbook,playbook-sat,resource-optimization,malware-detection,pinakes,assisted-installer,runtimes-java-general
+- name: INGRESS_VALID_UPLOAD_TYPES
+  value: advisor,compliance,hccm,qpc,rhv,tower,leapp-reporting,buckit,xavier,mkt,playbook,playbook-sat,resource-optimization,malware-detection,pinakes,assisted-installer,runtimes-java-general,openshift
```
Comment (gmcculloug) [inline deploy/clowdapp.yaml:141]: Do we still need `buckit` here with the addition of `openshift`?  Not sure sure, so just asking.

**Comment by gmcculloug** [comment]
@SteveHNH Would you be able to give this a quick review?  The env changes would also need to land in FRH.

File: README.md (hunk 1 above)
Comment (dehort) [reply README.md:34]: :+1:

File: deploy/clowdapp.yaml (hunk 2 above)
Comment (dehort) [reply deploy/clowdapp.yaml]: I think we can safely remove `buckit` because the code was mapping `openshift` upload type to the `buckit` topic.

**Comment by dehort** [comment]
I want to make sure we squash this before merging (please).

File: internal/config/config.go [hunk 3]
```diff
@@ -217,7 +217,7 @@ func Get() *IngressConfig {
 			KafkaTrackerTopic:     options.GetString("KafkaTrackerTopic"),
 			KafkaDeliveryReports:  options.GetBool("KafkaDeliveryReports"),
 			KafkaAnnounceTopic:    options.GetString("KafakAnnounceTopic"),
-			ValidTopics:           strings.Split(options.GetString("ValidTopics"), ","),
+			ValidUploadTypes:      strings.Split(options.GetString("Valid_Upload_Types"), ","),
```
Comment (skarekrow) [inline internal/config/config.go:220]: Maybe it's just me as I'm not a fan of these types of naming schemes :D, but does this look better than the paradigm we already have above? IE: `ValidUploadTypes`?

Comment (dehort) [reply internal/config/config.go:220]: It looks better to me!  ;)  With the existing approach, you end up with env vars that look like `INGRESS_VALIDUPLOADTYPES` (or `INGRESS_VALIDTOPICS` with the old name).  This is not very readable in my opinion.  I like to split the env var names on word boundaries.

**Pull request merged**